medusa/post_processor.py

```diff
@@ -731,26 +731,33 @@ def _get_ep_obj(self, show, season, episodes):
 
         return root_ep
 
-    def _get_quality(self, ep_obj):
+    def _quality_from_status(self, status):
+        """
+        Determine the quality of the file that is being post processed with its status.
+
+        :param status: The status related to the file we are post processing
+        :return: A quality value found in common.Quality
         """
-        Determine the quality of the file that is being post processed.
+        quality = common.Quality.UNKNOWN
+
+        if status in common.Quality.SNATCHED + common.Quality.SNATCHED_PROPER + common.Quality.SNATCHED_BEST:
```

Comment:
This should probably be removed, as it just assigns the quality of the current snatched status to the file that is being processed.